v21.2/datadog.md

```diff
@@ -90,7 +90,9 @@ Log collection is disabled by default in the Datadog Agent. Follow the steps in
 
 ## Step 3. Verify Datadog connection to CockroachDB
 
-Run the Datadog [`status` subcommand](https://docs.datadoghq.com/agent/guide/agent-commands/?tab=agentv6v7#agent-information) for your platform to verify that the CockroachDB integration is properly connected.
+To apply your configuration changes, ensure that you have restarted the Datadog Agent.
+
+Run the Datadog Agent [`status` subcommand](https://docs.datadoghq.com/agent/guide/agent-commands/?tab=agentv6v7#agent-information) for your platform to verify that the CockroachDB integration is properly connected.
```

Comment:
Just a note: here "integration" is not capitalized, as part of "CockroachDB Integration" in the Step 1 title it is capitalized — is this a specific tool?